README.md

```diff
@@ -167,6 +167,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [C# Dersleri](https://www.azkod.com/csharp)
 - :movie_camera: [C# Dersleri](https://www.youtube.com/watch?v=VytV9w2dAFs&list=PLKnjBHu2xXNPkeQtMOJczzEO6LK5OV35K) - [Murat Yücedağ](https://www.youtube.com/channel/UCbkbOlw8snP93RJ2BhH44Qw)
 - :movie_camera: [5 Saatlik C# Eğitimi](https://www.youtube.com/watch?v=2EkMrrX9sYY) - [Engin Demiroğ](https://www.youtube.com/channel/UCRjiquPh4mjPNoOV9eCilXQ)
+- :movie_camera: [Codecademy "Learn C#" Eğitimi [9 Bölüm]](https://youtu.be/0X7bEBSxygg) - [Soner Gönül](https://www.youtube.com/c/sonergonul)
```

Comment:
Ilk videonun linkini paylasmak yerine bu seriye ait videolari bir playliste koyup, playlistin linkini buraya koymak daha guzel olacaktir.

Reply:
Selam @lacivert, dediğin değişikliği yapıp tekrar commit ettim.